third_party/pkg/mdc_web_sass/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2019, the Dart project authors. All rights reserved.
```

Comment:
The mdc-web-components source is MIT licensed:

https://github.com/material-components/material-components-web/blob/master/LICENSE

We should follow suit here rather than using the traditional Dart license file.

Reply:
👍 

Technically, you can redistribute under another license:

> Can I redistribute MIT-licensed code under another license?
Yes. You cannot "re-license" MIT-licensed code, and the MIT-licensed code will remain under the MIT License, but you can redistribute it as part of another project under any license you wish. You should make it clear that any changes that you make to the MIT-licensed code are released under your new license. The MIT License text & copyright notice must still be included with any unmodified MIT-licensed code.

But since it requires the MIT License text, it's simpler to just use the MIT license